build.gradle

```diff
@@ -31,9 +31,14 @@ dependencies {
 
     implementation group: 'com.github.bibsysdev', name: 'apigateway', version: nvaCommonsVersion
     implementation group: 'com.github.bibsysdev', name: 'core', version: nvaCommonsVersion
+    implementation group: 'com.github.bibsysdev', name: 'lambdaauthorizer', version: nvaCommonsVersion
+    implementation group: 'com.github.bibsysdev', name: 'secrets', version: nvaCommonsVersion
+
 
     implementation group: 'com.amazonaws', name: 'aws-lambda-java-events', version: '2.2.7'
     implementation group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.2.0'
+    implementation group: 'com.amazonaws', name: 'aws-java-sdk-secretsmanager', version: '1.11.1000'
```

Comment:
we should probably update the version of nva-commons, in this project. In the latest commons secrets reader uses skd2.

Reply:
Gjør dette i ny task